x-pack/plugins/apm/public/components/app/ServiceMap/Controls.tsx

```diff
@@ -74,19 +78,19 @@ export function Controls() {
     if (cy) {
       const eles = cy.nodes();
       cy.animate({
-        ...animationOptions,
+        ...getAnimationOptions(theme.eui),
```

Comment:
I think you should pass in `theme` so we can use `EuiTheme` type consistently

```suggestion
        ...getAnimationOptions(theme),
```